proto/umee/leverage/v1/leverage.proto

```diff
@@ -139,4 +139,13 @@ message Token {
   // and enable_borrow set to false. Such tokens can be safely removed from the
   // oracle and price feeder as well.
   bool blacklist = 14;
+
+  // Maximum allowed collateral supply specifies how much of the token can
+  // be provided in total as a collateral in the system.
+  // Value is provided in pre cent, Allowed values are in [0; 100] range.
+  // 100 means that the token has no restriction. 10 means maximum 10% of total
```

Comment:
In cases like this, we usually go for an `sdk.Dec` ranging from `0.0` to `1.0`.

Reply:
I was thinking about it. Personally for small, well defined values I prefer ints:
* faster (both serialization / deserialization and computation)
* smaller in storage

Reply:
I think we kind of decided to use ints. Let's use int here - this is a new code, so doesn't change nor interact with other things. Really, no need to duplicate the work now.

In a separate PR I will update other relevant messages to use ints